org.openscad.OpenSCAD.yaml

```diff
@@ -3,8 +3,10 @@ runtime: org.kde.Platform
 runtime-version: 5.15-23.08
 sdk: org.kde.Sdk
 command: openscad
-rename-desktop-file: openscad-nightly.desktop
 rename-icon: openscad-nightly
+rename-mime-file: openscad.xml
```

Comment:
there is also a rename-mime-icon like used on the master branch. Should replace line 206 (deleted)

Reply:
Argh, :man_facepalming:. I did not have the master branch updated to the latest changes.

Locally I'm not getting a "missing screenshots" error which is a bit confusing again, lets see what the official builder has to say to that (all screenshots are remote URLs right now).